student-store-ui/src/components/App/App.jsx

```diff
@@ -10,6 +11,42 @@ import ProductDetail from "../ProductDetail/ProductDetail";
 import NotFound from "../NotFound/NotFound";
 
 export default function App() {
+
+  const [product, setProducts] = useState([]);
+  const [isFetching, setIsFetching] = useState(false);
+  const [error, setError] = useState(null);
+  const [isOpen, setIsOpen] = useState(false);
+  const [shoppingCart, setShoppingCart] = useState([]);
+  const [checkoutForm, setCheckoutForm] = useState({});
+
+  const API_URL = "https://codepath-store-api.herokuapp.com/store"
```

Comment:
this doesn't change you can move it outside the component, or on another file as a constant, just `https://codepath-store-api.herokuapp.com`